lib/Echidna/Mutator/Corpus.hs

```diff
@@ -40,7 +40,7 @@ mutator Deletion = deleteRandList
 selectAndMutate :: MonadRandom m
                 => ([Tx] -> m [Tx]) -> Corpus -> m [Tx]
 selectAndMutate f ctxs = do
-  rtxs <- weighted $ map (\(i, txs) -> (txs, fromInteger i)) $ DS.toDescList ctxs
+  rtxs <- weighted $ map (\(i, txs) -> (txs, fromIntegral i)) $ DS.toDescList ctxs
```

Comment:
Why this was changed?

Reply:
I changed `Integer` to `Int` in the Corpus type `type Corpus = Set (Int, [Tx])` as we don't need an unbounded Integer there.

Reply:
Great